pages/_app.tsx

```diff
@@ -14,11 +15,16 @@ import '@rainbow-me/rainbowkit/styles.css';
 import 'styles/globals.css';
 import XmtpProvider from '../contexts/XmtpProvider';
 
-import { alchemyProvider } from 'wagmi/providers/alchemy';
+// import { alchemyProvider } from 'wagmi/providers/alchemy';
+
+// const { chains, provider, webSocketProvider } = configureChains(
+//   [polygon],
+//     [alchemyProvider({ apiKey: process.env.NEXT_PUBLIC_ALCHEMY_API_KEY! })],
+// );
 
 const { chains, provider, webSocketProvider } = configureChains(
   [polygon],
-  [alchemyProvider({ apiKey: process.env.NEXT_PUBLIC_ALCHEMY_API_KEY! })]
+  [publicProvider()]
```

Comment:
Don't forget to return alchemy for prod.

Reply:
done